src/applications/disability-benefits/781/config/form.js

```diff
@@ -0,0 +1,52 @@
+// import _ from 'lodash/fp';
+
+// Example of an imported schema:
+// import fullSchema from '../21-0781-schema.json';
+
+import IntroductionPage from '../containers/IntroductionPage';
+import ConfirmationPage from '../containers/ConfirmationPage';
+
+// const { } = fullSchema.properties;
+
+// const { } = fullSchema.definitions;
+
+import { introductionText } from '../helpers';
+
+// Define all the fields in the form to aid reuse
+// const formFields = {};
+
+const formConfig = {
+  urlPrefix: '/',
+  submit: () =>
+    Promise.resolve({ attributes: { confirmationNumber: '123123123' } }),
+  trackingPrefix: 'ptsd-0781-0781a-',
+  introduction: IntroductionPage,
+  confirmation: ConfirmationPage,
+  formId: '1234',
+  version: 0,
+  prefillEnabled: true,
+  savedFormMessages: {
+    notFound: 'Please start over to apply for benefits.',
+    noAuth: 'Please sign in again to continue your application for benefits.',
+  },
+  title: 'Apply for increased disability compensation',
+  chapters: {
+    introductionPage: {
+      pages: {
+        ptsdIntroduction: {
```

Comment:
@dmccurdybah Lets move this into the pages directory, to keep the formConfig smaller